datafusion/datasource-parquet/src/file_format.rs

```diff
@@ -465,45 +465,103 @@ impl FileFormat for ParquetFormat {
     }
 }
 
-/// Coerces the file schema if the table schema uses a view type.
-pub fn coerce_file_schema_to_view_type(
+/// Apply necessary schema type coercions between table and file schemas
+///
```

Comment:
Perhaps clarify top line of documentation to say "Apply necessary schema type coercions to make file schema match table schema."  "Between" implies function will perform conversion both ways.

Reply:
thank you, updated